app/models/work-item.ts

```diff
@@ -121,7 +121,9 @@ export async function getNextWorkItem(
           .on(`${WorkflowStep.table}.stepIndex`, `${WorkItem.table}.workflowStepIndex`)
           .on(`${WorkflowStep.table}.jobID`, `${WorkItem.table}.jobID`);
       })
-      .where({ 'work_items.serviceID': serviceID, status: WorkItemStatus.READY })
+      .join(Job.table, `${WorkItem.table}.jobID`, '=', `${Job.table}.jobID`)
+      .where({ 'work_items.serviceID': serviceID, 'work_items.status': WorkItemStatus.READY })
+      .whereIn('jobs.status', [JobStatus.RUNNING, JobStatus.ACCEPTED])
```

Comment:
This clause seems possibly redundant, given where things are in the if-statement, and I could see it breaking things in the future if we ever added another job status.